docs/_include/variable_hydro.rst

```diff
@@ -0,0 +1,113 @@
+.. _user-advanced-features-variable-hydro:
+
+Overview
+""""""""""
+
+Variable Hydrodynamics is an advanced feature that enables users to change the 
+*state* of their device during simulation. In this context, a device *state* 
+can be related to any number of scenarios, such as a variable geometry 
+changing shape, a flooding body, a change in operational depth, load shedding 
+capabilities, time-dependent changes to a device, etc. A signal, such as 
+kinematics, dynamics, or time,  is used to alter the state of the device
+during simulation. This feature expands WEC-Sim's simulation capabilities and enables
+modeling a new breadth of scenarios in the time domain.
+
+Different states of a device are represented by different sets of boundary 
+element method data. User defined logic and user selected signals determine
+when the state changes. The Variable Hydrodynamics feature does not determine
+a specific scenario, state, signal, or discretization required. It is the 
+user's responsibility to implement a specific variable hydrodynamics case
+using best practices and validate the results with higher fidelity data.
+
```

Comment:
This section is that first one that users will read. I recommend simplifying it to clearly describe what/how variable hydro works, and then provide an example. Here's my initial attempt:

Variable hydrodynamics allows WEC-Sim bodies to simulate bodies with hydrodynamic data from multiple BEM runs for a single body. Hydrodynamic data from each BEM run is parsed into H5 files, which are then initialized in the instance of the body class, for example `body(1) = bodyClass({'H5FILE_1.h5','H5FILE_2.h5',...);`. This feature allows users to simulate bodies with different initial conditions, for example body that is floating or submerged. Users then specify the condition to switch when each H5 is used. The condition must be based on a WEC-Sim state...

Reply:
@akeeste I used ChatGPT to refine the overview and here's the suggestion:

## Overview

Variable Hydrodynamics enables WEC-Sim bodies to utilize hydrodynamic data from multiple BEM runs within a single body instance. Each BEM run's data is stored in H5 files, which are initialized in the body class, for example:
  
`body(1) = bodyClass({'H5FILE_1.h5', 'H5FILE_2.h5', ...});`
  
 This feature allows users to simulate different initial conditions, such as floating or submerged states. Users specify the conditions that trigger the use of each H5 file, based on a WEC-Sim state.

Reply:
I agree this text could be more concise, I'll revise this text. 

The ChatGPT response is certainly concise, bit its inaccurate when for some of the details (H5 files aren't being initialized, the feature is not about initial conditions)